frontend/src/components/annotation/image_and_video_annotation/annotation_core.vue

```diff
@@ -1733,6 +1738,30 @@ export default Vue.extend({
   },
   // TODO 312 Methods!! refactor in multiple files and classes.
   methods: {
+
+    setupHotkeys() {
+
```

Comment:
`HotkeyListener` API is a lot cleaner than we we're doing right now. Please compare it with commented out functions.